src/main/java/at/hannibal2/skyhanni/features/garden/GardenNextJacobContest.kt

```diff
@@ -315,6 +315,7 @@ object GardenNextJacobContest {
     private fun warn(timeInMillis: Long, crops: List<CropType>) {
         if (!config.warn) return
         if (config.warnTime <= timeInMillis / 1000) return
+        if (!warnForCrop()) return
```

Comment:
Why are you checking for negative here? Why don't you just return the opposite in warnForCrop() ?

Reply:
That would be kind of confusing
also this sort of thing is used for isenabled functions